tests/unit/test_tag_chartbeat.py

```diff
@@ -11,7 +11,7 @@
 
 from analytical.templatetags.chartbeat import ChartbeatTopNode, \
         ChartbeatBottomNode
-from analytical.tests.utils import TagTestCase
+from utils import TagTestCase
 from analytical.utils import AnalyticalException
```

Comment:
We should apply the correct sort order here. Grouping is suggested by Pylint as follows:

1. Standard library imports first, in alphabetical order
1. Other packages afterwards, alphabetical order again
1. Local imports (aka our own package) at the end

This comment is valid for all modules, of course.

Reply:
I'm strongly in favour of leaving this to `isort`, rather than sorting the importance manually.

We could make a follow-up PR for that, and maybe `black` for good measure too?

Reply:
Did `isort` produced the current sort order? With `utils` between `analytical.templatetags.*` and `analytical.utils`?

Weird.

Reply:
No, this just retains the previous sort order without any re-ordering.

Reply:
> We could make a follow-up PR for that, and maybe `black` for good measure too?

I'm happy to have a look at this.